websites/Y/YomuMangás/metadata.json

```diff
@@ -0,0 +1,72 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "293913134748401674",
+		"name": "Vitu™#2002"
+	},
+	"service": "YomuMangás",
+	"description": {
+		"en": "Read portuguese mangas and doujinshis online for free without ads at YomuMangás",
+		"pt_BR": "Leia mangás e doujinshis, de graça e sem anúncios na YomuMangás."
+	},
+	"url": [
+		"yomumangas.com",
+		"dev.yomumangas.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/RBkqiAg.png",
+	"thumbnail": "https://i.imgur.com/EqYC4Qo.png",
+	"color": "#121212",
+	"category": "anime",
+	"tags": [
+		"anime",
+		"manga",
+		"doushinji",
+		"geek",
+		"nsfw"
+	],
+	"settings": [
+		{
+			"id": "images",
+			"title": "Mostrar Imagens (Avatars & Covers)",
+			"icon": "fad fa-images",
+			"value": true
+		},
+		{
+			"id": "manga",
+			"title": "Mostrar Manga",
+			"icon": "fad fa-book",
+			"value": true
+		},
+		{
+			"id": "chapter",
+			"title": "Mostrar Capítulo",
+			"icon": "fad fa-clone",
+			"value": true
+		},
+		{
+			"id": "user",
+			"title": "Mostrar Usuário",
+			"icon": "fad fa-user",
+			"value": true
+		},
+		{
+			"id": "timestamp",
+			"title": "Mostrar Timestamp",
+			"icon": "fad fa-stopwatch",
+			"value": true
+		},
+		{
+			"id": "adult",
+			"title": "Mostrar NSFW (Nome)",
+			"icon": "fad fa-exclamation-triangle",
+			"value": false
+		},
+		{
+			"id": "privacy",
+			"title": "Modo Privado",
+			"icon": "fad fa-user-secret",
+			"value": false
+		}
+	]
```

Comment:
What do each one of these do please?

Reply:
Mostrar Images/Show Images, this option when enabled show in the large image the current manga cover (if not adult content) and the avatar from the user that you are looking

Mostrar Mangá/Show Manga, this option when enabled show the current manga when you are reading or on the manga page, if disabled, "Lendo um Mangá"/"Reading a Manga"

Mostrar Capítulo/Show Chapter, has the same function of the "Show Manga", but with the reading chapter

Mostrar Timestamp/Show Timestamp just show the how much time the user are in the website

Mostrar NSFW/Show NSFW, when enabled it's show only the name of the current manga that you are reading

Modo Privado/Privacy Mode, this options make the code ignore all pages and only show that the user are in the website.